internal/commands/whoami/inputs.go

```diff
@@ -0,0 +1,14 @@
+package whoami
+
+import (
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/terminal"
+)
+
+type inputs struct {
```

Comment:
[style/nit] for such a simple struct, we don't need a separate file for this.  just define it all in the `command.go` file